.github/workflows/real_storage.yml

```diff
@@ -0,0 +1,75 @@
+      bucket:            {default: 'arcticdb-ci-test-bucket-01', type: string, description: The name of the S3 bucket that we will test against}
+      endpoint:          {default: 's3.eu-west-2.amazonaws.com', type: string, description: The address of the S3 endpoint}
+      region:            {default: 'eu-west-2', type: string, description: The S3 region of the bucket}
+jobs:
+  setup:
+    strategy:
+      matrix:
+        # Declaring the dummy fields here to aid the Github Actions linting in VSCode and to provide documentation
+        os: [0] # Decouples the steps from any distro version changes
+        distro: [0]
+        cmake_preset_prefix: [0]
+        cibw_build_suffix: [0]
+        envs: [0]
+        build_dir: [0] # Must be an absolute path
+        symbols: [0] # Glob for symbol symbol files. Used for including in follower builds and exclusion on others.
+        do_not_archive: [0]
+        test_services: [0] # Github service containers to spin up for the pytest run
+        exclude:
+            - os: 0
+        include:
+            - ${{fromJSON(inputs.matrix)[0]}} # The items after 0 are for tests only
+    runs-on: ${{matrix.distro}}
+    defaults:
+      run: {shell: bash}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Select Python (Linux)
+        if: matrix.os == 'linux'
+        run: echo /opt/python/${{env.python_impl_name}}*/bin >> $GITHUB_PATH
+
+      - name: Select Python (Windows)
+        if: matrix.os == 'windows'
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.${{inputs.python3}}"
+
+      - name: Install latest release
+        run: |
+          pip install arcticdb
+
+      - name: Set real storage variables
+        run: |
+          echo "ARCTICDB_REAL_STORAGE_TESTS=1" >> $GITHUB_ENV
+          echo "ARCTICDB_REAL_S3_BUCKET=${{ inputs.bucket }}" >> $GITHUB_ENV
+          echo "ARCTICDB_REAL_S3_ENDPOINT=${{ inputs.endpoint }}" >> $GITHUB_ENV
+          echo "ARCTICDB_REAL_S3_REGION=${{ inputs.region }}" >> $GITHUB_ENV
+          echo "ARCTICDB_REAL_S3_CLEAR=${{ inputs.clear }}" >> $GITHUB_ENV
+          echo "ARCTICDB_REAL_S3_ACCESS_KEY=${{ secrets.AWS_S3_ACCESS_KEY }}" >> $GITHUB_ENV
+          echo "ARCTICDB_REAL_S3_SECRET_KEY=${{ secrets.AWS_S3_SECRET_KEY }}" >> $GITHUB_ENV
+          echo "ARCTICDB_REAL_STORAGE_LIB_NAME=${{ github.ref_name }}_${{matrix.os}}_3_${{inputs.python3}}" >> $GITHUB_ENV
+        
+      - name: Seed the storage
+        if: inputs.job_type == 'seed'
+        run: |
+          echo "Running seed for ${{matrix.os}}_3.${{inputs.python3}}"
+          python build_tooling/seed_storage.py "${{matrix.os}}_3_${{inputs.python3}}"
+
+      - name: Verify the storage
+        if: inputs.job_type == 'verify'
+        run: |
+          echo "Running verify for ${{ github.ref_name }} on ${{matrix.os}}_3_${{inputs.python3}}"
+          python build_tooling/verify_storage.py "${{github.ref_name}}"
```

Comment:
I'm confused why we need this and also [test_real_storage.py](https://github.com/man-group/ArcticDB/pull/663/files#diff-3d0e73f7d27236996b7939f3b768f95a5fae6c95de5fa05e6686a242dcde9af0). It makes sense to have some checks that the seeding went OK, but why not fold them in to `seed_storage.py`?

Reply:
This is to verify the data that was written by the version that is being built, not in the seeding.
The writing is done in [test_real_storage.py](https://github.com/man-group/ArcticDB/pull/663/files#diff-3d0e73f7d27236996b7939f3b768f95a5fae6c95de5fa05e6686a242dcde9af0) in the test_real_s3_storage_write